tests/build/returnconst.pyx

```diff
@@ -0,0 +1,2 @@
+cdef const int f():
```

Comment:
I'll suggest moving it to `tests/run/return.pyx`. Mainly because there's a bigger overhead in a new test than there is in adding to an existing one.

Reply:
Thanks, moved